src/uwtools/drivers/ww3.py

```diff
@@ -0,0 +1,82 @@
+"""
+A library driver for ww3.
+"""
+
+from pathlib import Path
+from typing import List, Optional
+
+from iotaa import asset, task, tasks
+
+from uwtools.api.template import render
+from uwtools.drivers.driver import Assets
+from uwtools.strings import STR
+from uwtools.utils.tasks import file
+
+
+class WaveWatchIII(Assets):
+    """
+    A library driver for ww3.
+    """
+
+    def __init__(
+        self,
+        config: Optional[Path] = None,
+        dry_run: bool = False,
+        batch: bool = False,
+        key_path: Optional[List[str]] = None,
+    ):
+        """
+        The driver.
+
+        :param config: Path to config file (read stdin if missing or None).
+        :param dry_run: Run in dry-run mode?
+        :param batch: Run component via the batch system?
+        :param key_path: Keys leading through the config to the driver's configuration block.
+        """
+        super().__init__(config=config, dry_run=dry_run, batch=batch, key_path=key_path)
+
+    # Workflow tasks
+
+    @task
+    def namelist_file(self):
+        """
+        The namelist file.
+        """
+        fn = "ww3_shel.nml"
```

Comment:
I know that this is the name of the namelist file that we decided to initially support, but is this the filename that WW3 expects to actually see in the run directory? Is there not maybe a canonical filename that the application will read, with the user expected to copy their file (perhaps starting from one of the example files provided with the application) into the run directory with that canonical name?

Reply:
Strictly, its expecting `ww3_shel.nml` or `ww3_shel.inp` but that's what WW3 is expecting to see

Reply:
All good, then.